drivers/oled/oled_driver.c

```diff
@@ -555,7 +564,13 @@ bool oled_off(void) {
         return !oled_active;
     }
 
-    static const uint8_t PROGMEM display_off[] = {I2C_CMD, DISPLAY_OFF};
+    static const uint8_t PROGMEM display_off[] =
+#ifdef OLED_DRIVER_DISPLAY_FADE
+        {I2C_CMD, FADE_BLINK, 0x20 | OLED_DRIVER_DISPLAY_FADE_INTERVAL};
```

Comment:
Please add a constant for '0x20' so we know this is the "Enable Fade Out mode." flag.
Additionally, might be useful to add beside that define, a '0x30' define for blinking.

Reply:
Do you have any preferences for naming? I was thinking something like this, but it's not that obvious that the `ENABLE_FADE` and `ENABLE_BLINK` are supposed to follow the `FADE_BLINK` command: 

```
// Advance Graphic Commands
#define FADE_BLINK 0x23
#define ENABLE_FADE 0x20
#define ENABLE_BLINK 0x30
```

```
{I2C_CMD, FADE_BLINK, ENABLE_FADE | OLED_DRIVER_DISPLAY_FADE_INTERVAL};
```

Reply:
Ya, that looks fine to me.